### DISCUSSION TITLE
Form Field Validation in real time for good user experience

### DISCUSSION THREAD

**Discussion by jay-p-b-7span** [directus/directus#16815] [Archive]
Problem -> Currently on Login screen , we do not have any field value checking when user hits login.

Expectation -> It would be nice if we have a validation library setup in project for validating all the fields in project in real time for good user experience.  Let me know whether to implement this feature @rijkvanzanten . 

Examples 
![image](https://user-images.githubusercontent.com/93135924/207505411-f53d1718-48e9-4b94-a46a-751c5953a156.png)

![image](https://user-images.githubusercontent.com/93135924/207505449-0a3d3bcf-798c-4d10-ad09-3af099cd8cbe.png)

Ending Note -> I am thankful to the team of directus for tihs wonderful project.

**Comment by paescuj** [comment]
Just to be sure: Are you only talking about the login form or other places? In case of the login form validation is already in place thanks to the `type="email"` property on the input field which should be supported in modern browsers:

<img width="385" alt="Screenshot 2022-12-14 at 11 19 26" src="https://user-images.githubusercontent.com/5363448/207570327-690ca385-cf9a-472c-909a-5ec36b38439c.png">
